### PR TITLE
[JS] feat: handoff activity handler

### PR DESCRIPTION
**Pull request by aacebo**
## Linked issues

#minor
#1447 

## Details

add activity handler for handoffs

### CODE REVIEW

File: js/packages/teams-ai/src/Application.ts [hunk 1]
```diff
@@ -609,7 +609,29 @@ export class Application<TState extends TurnState = TurnState> {
...
             return handler(context, state, context.activity.value as O365ConnectorCardActionQuery);
         };
-        this.addRoute(selector, handlerWrapper);
+        this.addRoute(selector, handlerWrapper, true);
+        return this;
+    }
+
+    /**
+     * Registers a handler to process when a file consent card is accepted by the user.
+     * @param {(context: TurnContext, state: TState, continuation: string) => Promise<void>} handler Function to call when the route is triggered.
```
Comment (corinagum) [inline js/packages/teams-ai/src/Application.ts:618]: What is continuation string?

Comment (corinagum) [reply js/packages/teams-ai/src/Application.ts:618]: Is there a link to official docs we can link to ?

File: python/packages/ai/teams/app.py [hunk 2]
```diff
@@ -512,6 +512,49 @@ async def __handler__(context: TurnContext, state: StateT):
...
         return __call__
 
+    def handoff(
+        self,
+    ) -> Callable[
+        [Callable[[TurnContext, StateT, str], Awaitable[None]]],
+        Callable[[TurnContext, StateT, str], Awaitable[None]],
+    ]:
+        """
+        Registers a handler for when a O365 connector card action is received from the user.
```
Comment (lilyydu) [inline python/packages/ai/teams/app.py:522]: do you mind updating this method signature

**Review by corinagum — APPROVED** [review]
JS LGTM

**Review by lilyydu — APPROVED** [review]

**Pull request merged**